.github/workflows/update-contributors.yml

```
@@ -51,5 +51,5 @@ jobs:
           title: 'chore: update contributors list'
           body: 'Automatically updated contributors list using Git log'
           branch: update-contributors
-          base: main # Use the main branch as the base
-          delete-branch: true
+          base: main  # Use the main branch as the base
+          delete-branch: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)